converter.go

```diff
@@ -94,9 +94,13 @@ func attrToSentryEvent(attr slog.Attr, event *sentry.Event) {
 			event.User.Data["request"] = anyValueToString(v)
 		}
 	} else if kind == slog.KindGroup {
-		event.Extra[attr.Key] = attrToMap(attr.Value.Group())
+		event.Contexts[attr.Key] = attrToMap(attr.Value.Group())
 	} else {
-		event.Extra[attr.Key] = attr.Value.Any()
+		// "context" should not be added to underlying context layers (see slog.KindGroup case).
+		if _, ok := event.Contexts["context"]; !ok {
```

Comment:
You _could_ make "context" a public variable, optionally changeable by library users. (To stop overlapping Context names for instance)

Reply:
Would you prefer "extra" ?